devbox/src/cmdproxy/ProxyServer.scala

```diff
@@ -37,7 +32,9 @@ object Response {
  * Response:
  *   {"exitCode":0,"output":"bla bla"}
  */
-class ProxyServer(dirMapping: Seq[(os.Path, os.RelPath)], port: Int = ProxyServer.DEFAULT_PORT)(implicit logger: FileLogger) {
+class ProxyServer(dirMapping: Seq[(os.Path, os.RelPath)],
```

Comment:
Can you update the scaladoc here with your changes to the protocol? I think it may help later.

Reply:
yeah i'll update it